STYLEGUIDE.mdx

```diff
@@ -38,14 +38,14 @@ and some JavaScript code (check `.eslintrc.js`) is using Prettier to format code
 can run `node scripts/eslint --fix` to fix linting issues and apply Prettier formatting.
 We recommend you to enable running ESLint via your IDE.
 
-Whenever possible we are trying to use Prettier and linting over written style guide rules.
+Whenever possible we are trying to use Prettier and linting overwritten style guide rules.
```

Comment:
I actually think the spacing was intentional here 🙂 

But it might be clearer to say:
```suggestion
Whenever possible we are trying to use Prettier and linting instead of maintaining a set of written style guide rules.
```

Reply:
good suggestion, done!